sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -345,4 +345,131 @@
+    <item>
+      <EFDB_Name>timestamp</EFDB_Name>
+      <Description>Starting time of data acquisition (TAI unix seconds).</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>second</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>interval</EFDB_Name>
+      <Description>Data collection time interval. N-th (starting from 0) sample is assumed to be taken at "timestamp" + ("interval"/200) * N.</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>second</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>accelerationX</EFDB_Name>
+      <Description>Acceleration in x direction.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>m/s2</Units>
+      <Count>200</Count>
+    </item>
+    <item>
+      <EFDB_Name>accelerationY</EFDB_Name>
+      <Description>Acceleration in y direction.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>m/s2</Units>
+      <Count>200</Count>
+    </item>
+    <item>
+      <EFDB_Name>accelerationZ</EFDB_Name>
+      <Description>Acceleration in Z direction.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>m/s2</Units>
+      <Count>200</Count>
+    </item>
+    <item>
+      <EFDB_Name>location</EFDB_Name>
+      <Description>Physical location of the sensor (e.g. AuxTel Truss).</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+  </SALTelemetry>
+  <SALTelemetry>
+    <EFDB_Topic>ESS_accelerometerPSD</EFDB_Topic>
+    <Description>PSD data from a 3-axis accelerometer.</Description>
+    <item>
+      <EFDB_Name>sensorName</EFDB_Name>
+      <Description>The user-defined name of the accelerometer (e.g. AuxTel_accel1).</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>timestamp</EFDB_Name>
+      <Description>Starting time of data acqusition (TAI Unix seconds) used to compute PSDs.</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>second</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>interval</EFDB_Name>
+      <Description>Time interval for which PSDs were collected.</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>second</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>startFrequency</EFDB_Name>
+      <Description>PSD start frequency.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>Hz</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>endFrequency</EFDB_Name>
+      <Description>PSD start frequency.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>Hz</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>numberOfSamples</EFDB_Name>
+      <Description>Number of samples used. Array elements with index greater than numberOfSamples are invalid and can contain any value (0 preferred).</Description>
```

Comment:
I don't quite understand this.
This is not the number of samples used in calculating the PSD, correct?
maybe call it, "validPSDDataPoints" -- if you mean that between min and max frequencies then only ~10 values make sense.

Reply:
"numValues" is fine with me, then just change "samples" to "values" in the description of course.

Reply:
<EFDB_Name>validPSDDataPoints</EFDB_Name>
      <Description>Number of sampled frequencies. PSD array elements with index greater than validPSDDataPoints are invalid and can contain any value (0 preferred).</Description>

should be better?